module/ColorJudge.cpp

```diff
@@ -0,0 +1,55 @@
+/**
+ * @file ColorJudge.cpp
+ * @brief 色識別クラス
+ * @author hiroto0927, miyashita64
+ */
+
+#include "ColorJudge.h"
+
+ColorJudge::ColorJudge() {}
+
+// RGBをHSVに変換し、色を識別する
+COLOR_NUMBER ColorJudge::getColorNumber(rgb_raw_t& rgb)
```

Comment:
```suggestion
COLOR_NUMBER ColorJudge::getColorNumber(rgb_raw_t const& rgb)
```

参照渡しだと、値の書き換えが可能になってしまう。
書き換える必要がない場合は、constを付けて書き換えを不可にすることで、呼び出し側は安心して呼び出せる。
...らしい。
https://qiita.com/agate-pris/items/05948b7d33f3e88b8967